src/development/LocalTest/Services/Profile/Implementation/UserProfilesWrapper.cs

```diff
@@ -27,16 +30,24 @@ public UserProfilesWrapper(IOptions<LocalPlatformSettings> localPlatformSettings
         /// <inheritdoc />
         public async Task<UserProfile> GetUser(int userId)
         {
-            UserProfile user = null;
             string path = this._localPlatformSettings.LocalTestingStaticTestDataPath + "Profile/User/" + userId + ".json";
-            if (File.Exists(path))
+            if (!File.Exists(path))
```

Comment:
This if must be inverted again. It's now skipping all files that exists. The method must also return the identified user profile. It's now returning null in all cases.

Reply:
😠